std/internal/test/uda.d

```diff
@@ -14,3 +14,13 @@ struct HasPrivateMembers
   @Attr private int c;
   private int d;
 }
+
+// If getSymbolsByUDA is mixed into the same scope it also returns private members
```

Comment:
It would be nice if we could get this in the docs somehow. Should the doc comment for `getSymbolsByUDA` at least mention that it doesn't access private members unlesss you mix it in?